src/bentoml/_internal/configuration/v1/default_configuration.yaml

```diff
@@ -47,17 +72,47 @@ api_server:
     metrics:
       host: 0.0.0.0
       port: 3001
+  tracing:
```

Comment:
let's move this out of api_server. Tracing has to be shared between runners and api servers.

Reply:
Do we parse these options to runner tracing?